lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -73,7 +83,7 @@ export async function updateArtifacts({
           'gradlew',
           'gradlew.bat',
         ].map(async fileProjectPath =>
-          addIfUpdated(status, projectDir, fileProjectPath)
+          addIfUpdated(config, status, fileProjectPath)
```

Comment:
```suggestion
          addIfUpdated(status, fileProjectPath)
```

Reply:
fixed in https://github.com/renovatebot/renovate/pull/5876/commits/e307e2da378e9d02cfcbdba2b01c7593f79a10e2